apps/meteor/app/api/server/lib/rooms.ts

```diff
@@ -181,7 +182,7 @@ export async function findRoomsAvailableForTeams({ uid, name }: { uid: string; n
 		Subscriptions.findByUserIdAndRoles(uid, ['owner'], { fields: { rid: 1 } }).fetch() as Pick<ISubscription, 'rid'>[]
 	).map((item) => item.rid);
 
-	const rooms = await Rooms.findChannelAndGroupListWithoutTeamsByNameStartingByOwner(uid, name, userRooms, options).toArray();
+	const rooms = await Rooms.findChannelAndGroupListWithoutTeamsByNameStartingByOwner(name, userRooms, options).toArray();
```

Comment:
why did you remove it? i'm quite sure uid is important here

Reply:
I thought the uid was not being used inside the function, I'll double check